pkg/agent/openflow/pipeline.go

```diff
@@ -1766,17 +1848,32 @@ func (f *featureNetworkPolicy) conjunctionActionDenyFlow(conjunctionID uint32, t
 		Done()
 }
 
-func (f *featureNetworkPolicy) conjunctionActionPassFlow(conjunctionID uint32, table binding.Table, priority *uint16, enableLogging bool) binding.Flow {
+func (f *featureNetworkPolicy) conjunctionActionPassFlow(conjunctionID uint32, table binding.Table, priority *uint16,
+	enableLogging, isIngress, isIGMP bool) binding.Flow {
 	ofPriority := *priority
 	conjReg := TFIngressConjIDField
 	nextTable := IngressRuleTable
 	tableID := table.GetID()
-	if _, ok := f.egressTables[tableID]; ok {
-		conjReg = TFEgressConjIDField
-		nextTable = EgressRuleTable
+	if f.isMulticastIngressRule(tableID, isIngress, isIGMP) || f.isIGMPEgressRule(tableID, isIngress, isIGMP) {
+		return nil
 	}
-	flowBuilder := table.BuildFlow(ofPriority).MatchConjID(conjunctionID).
+	if f.isMulticastEgressRule(tableID, isIngress, isIGMP) {
+		// todo: confirm is multicast has pass
+		//return nil
```

Comment:
?

Reply:
removed, as multicast NP does not supprt Pass and Reject.